src/core/hid/hid_types.h

```diff
@@ -256,6 +256,8 @@ enum class NpadStyleSet : u32 {
     Lager = 1U << 11,
     SystemExt = 1U << 29,
     System = 1U << 30,
+    All = Fullkey + Handheld + JoyDual + JoyLeft + JoyRight + Gc + Palma + Lark + HandheldLark +
```

Comment:
```suggestion
    All = Fullkey + Handheld | JoyDual | JoyLeft | JoyRight | Gc | Palma | Lark | HandheldLark |
```

etc for the other identifiers on the line below